services/keywords/extract.py

```diff
@@ -33,6 +34,7 @@ def extract_keywords(text):
         the lemma and part of speech for said keywords
     """
     gcloud_response = gcloud_syntax_extraction(text)
+    logging.info("gcloud syntax response:", gcloud_response)  # pylint: disable=logging-too-many-args
```

Comment:
You're getting a linter error because this is an incorrect usage of `logging.info`. You need to specify a format string: https://docs.python.org/3.8/library/logging.html#logging.debug

So this should be:
```python
logging.info("gcloud syntax response: %s", gcloud_response)
```

Reply:
Wow that's a pretty limited selection of string interploration. Not even f string interploration works with pylint